applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h

```diff
@@ -60,6 +60,20 @@ int bt_rend_volume_unmute(void);
  */
 int bt_rend_discover(struct bt_conn *conn);
 
+/**
+ * @brief	Get advertising data from this module.
+ *
+ * @note	This partial data is used to build a complete extended advertising packet.
+ *
+ * @param [in/out] uuid_buf Buffer being populated with UUIDs.
+ * @param [in/out] adv_buf Buffer being populated with ext adv elements.
+ * @param [in] adv_buf_vacant Number of vacant elements in @p adv_buf.
+ *
+ * @return	Negative values for errors or num elements added to adv_buf.
```

Comment:
```suggestion
 * @return	Negative values for errors or num elements added to @p adv_buf.
```

This is the same for all of the XXX_adv_get() funcions